linux/CMakeLists.txt

```diff
@@ -1 +1,7 @@
+install(FILES org.qgis.qgis.desktop DESTINATION share/applications)
 install(FILES org.qgis.qgis.appdata.xml DESTINATION share/metainfo)
+install(FILES ${CMAKE_SOURCE_DIR}/images/icons/qgis_icon.svg RENAME qgis.svg DESTINATION share/icons/hicolor/scalable/apps)
+
+foreach(_type qgs qlr qml qpt)
+  install(FILES ${CMAKE_SOURCE_DIR}/images/icons/qgis_${_type}_icon.svg DESTINATION share/icons/hicolor/scalable/mimetypes/)
+endforeach()
```

Comment:
Similar problem as we had above: the `.desktop` file expects:

`qgis-${_type}.svg` and _not_ `qgis_${_type}_icon.svg`

I also have to say that it looks like nautilus does not play very well with mime icons in svg. Fixing the issue mentioned above I get an ugly rendered icon in nautilus (Fedora 28, up-to-date):
![screenshot from 2018-06-25 10-46-54](https://user-images.githubusercontent.com/1818657/41840187-68b921cc-7865-11e8-9074-60a91d5963db.png)

I'm also worried about systems running simple DE/WM so, as a general comment, I think we must keep the PNGs files as they were.